packages/api/src/controllers/RateController.js

```diff
@@ -0,0 +1,28 @@
+const Kucoin = require('../kucoin')
+const Coinbase = require('../coinbase')
+
+class RateController {
+  async getUSDRate (request, response) {
+    const kucoinResponse = await Kucoin.getUSDRate().catch((err) => {
+      console.error(err)
+      return null
+    })
+
+    if (typeof kucoinResponse === 'number') {
+      response.send({ usd: kucoinResponse, source: 'Kucoin' })
```

Comment:
I believe, you are missing a return here. because in case kucoin succesully returned rates, it goes next to the coinbase and will invoke response.send() twice